dbms/src/Core/Settings.h

```diff
@@ -75,6 +75,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingUInt64, background_schedule_pool_size, 16, "Number of threads performing background tasks for replicated tables. Only has meaning at server startup.") \
     \
     M(SettingMilliseconds, distributed_directory_monitor_sleep_time_ms, 100, "Sleep time for StorageDistributed DirectoryMonitors in case there is no work or exception has been thrown.") \
```

Comment:
Need to document the logic of exponential backoff.

Reply:
Good point, added